java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -492,17 +511,32 @@ public void message(LocoNetMessage m) {
      * Collect data from specific slots
      */
     void checkSpecialSlots(LocoNetMessage m, int slot) {
-        // TODO: Collect CS opswitch
+        if (!pmManagerGotReply &&
```

Comment:
It seems awkward to sequence a `SlotManager` operation based on a `PowerManager` operation which invokes a `SlotManager` method.  What other mechanisms have you considered?

Reply:
I Tried a timer, like the slotscan start up, but could not get it to work. The power interogate already probes slot zero, so when that first slot message comes back we know we have a command station powered up, even if track power is off. If I just used a timer then if you connect via a pr3, the message is going out without any knowledge that the command station is up.  This seemed the best approach.

Reply:
I was thinking something more like assuming that the power manager will perform the slot 0 read, and paying attention to slot number when a slot read report is received.  If the slot number is 0, then set the flag.  With such a mechanism, it would not be necessary for power manager to fiddle with what is effectively a slot management function.

Reply:
I must admit that I have never comprehensively studied computer-to-LocoNet interface behaviors when JMRI is started before LocoNet is "alive".  I know that at least _some_ will "queue" one or more requests, and then send any queued request once LocoNet becomes usable.

Reply:
Only looking for slot zero, and only if the CS reports slot 250. In the longer term I think maybe we could gatherup all this "probing" into one place and allow it to be managed.